Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:14
+FROM docker.io/node:14
 
-COPY package*.json ./
-RUN npm install
+COPY ./app /app
+
+USER root
+
+RUN set -exu \
+  && groupadd --gid 1101 fishy \
+  && useradd --uid 1101 --gid 1101 --create-home --shell /sbin/nologin fishy \
```

Comment:
Why are you using `1101` as the group and user ID?

Reply:
Just a number that has a low chance of overlapping with anything on the host - not that overlapping with host uid/gid would matter, just the principle of the thing.

We could pick anything, `1101` was selected because it's `1000 + 101`: 100 above the "normal" user id starting point + 1.